README.md

```diff
@@ -1 +1,144 @@
+
+To do list is a project meant to master my skills in version control, GitHub flow, webpack and JavaScript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML & CSS
+- Linters
+- Git & Github
+- webpack
+- JavaScript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Pop up window]**
+- **[contact form storage capability]**
+- **[addition of logo rotate on hover]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## Getting Started
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://kel-nana.github.io/To-Do-list/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+**This is an example of how you may give instructions on setting up your project locally.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Git in your local machine
+- Code editor (VS code, Atom, etc)
+- Terminal
+
+### Setup
+
+- git clone git@github.com:Kel-nana/To-Do-list.git
+
+- cd To-Do-list
+
+### Usage
+
+- Add the changes you want and feel free to use in your project!
+
+### Run tests
+
+- Check that all the tests are correctly passed🤝
+
+### Merge
+
+- Merge it so we can all see your work!!🤝
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@Kel-nana](https://github.com/Kel-nana)
+- Linkedin: [@kelvin-irungu](https://www.linkedin.com/in/kelvin-irungu-838923249/)
+- Twitter: [@kelvin-irungu](https://twitter.com/home)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Customise styling]**
+- [ ] **[add a comment section]**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Kel-nana/Purple-Coders---HTML-CSS-JavaScript-capstone-project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Share ideas on how to improve this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse documentation and my coding partner
+
+<p align="right">
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] I would suggest you please remove the `NOTE` part as it does not make you README clean.
- [ ] I will provide a hint on how to use the MIT license. This is no the way how we make an MIT license. Please see below for more hints. This is the [Text file](https://choosealicense.com/licenses/mit/)
### Hint
![s](https://user-images.githubusercontent.com/81727988/223686175-a7f42099-488e-4b0c-8372-6503021d3e21.png)

Reply:
MIT [`license`](https://github.com/Kel-nana/To-Do-list/blob/498659cc651f8e8866fdacfe4b07d2f2fb174178/README.md?plain=1#L166)